src/repositories/QueueRepository/QueueRepository.ts

```diff
@@ -99,6 +99,43 @@ export class QueueRepository {
     });
   }
 
+  /**
```

Comment:
This big block should be removed. Probably just snuck in from the merge.

Reply:
Yeah. Fixed.